### DISCUSSION TITLE
About GPU usage and Temps

### DISCUSSION THREAD

**Discussion by JMnzsz** [34736384/genshin-fps-unlock#334] [Q&A]
I've been using this FPS Unlocker for a while, but I'm quite concerned about my GPU:
I'm currently playing with max game settings on 1080p @ 144Hz and my RTX 3070Ti gets between 80 ~ 88°C (176 ~ 191°F) just after a couple minutes playing. It sits around 85~92% usage.

Is this normal, though? That's quite hot for just 1080p tbh, it ain't even 1444p or 240Hz, so I'm wondering if there's something I have to tweak
My specs are: i7-12700K, RTX 3070Ti, 32GB RAM, Z690 MoBo. Currently no overclock and i7 is watercooled, so its fine.

**Comment by 34736384** [comment]
30 series cards are rated for a maximum temperature of 93c. If you think 80+ is too high then crank the fan to 100%. With decent airflow you should be able to get it back down to 50-60
